src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -147,11 +151,8 @@ JsonObject GlobalAppSettings::ToJson() const
 
     jsonObject.Insert(SHOW_TABS_IN_TITLEBAR_KEY,
                       JsonValue::CreateBooleanValue(_showTabsInTitlebar));
-    if (_requestedTheme != ElementTheme::Default)
-    {
         jsonObject.Insert(REQUESTED_THEME_KEY,

```

Comment:
would you mind fixing the indentation here, since you removed the enclosing scope?

Reply:
oh yes, right.